packages/scheduler/lib/scheduler.ts

```diff
@@ -50,6 +49,9 @@ export class Scheduler {
                 }
             });
             this.monitor.start();
+            this.scheduling = new SchedulingWorker({ databaseUrl: dbClient.url, databaseSchema: dbClient.schema });
+            // TODO: ensure there is only one instance of the scheduler
```

Comment:
It is on my list to have a simple distributed lease mechanism to ensure this is a single scheduling worker. For now it is enforced by the fact that orchestrator service is set to 1 instance and doesn't autoscale.